test/over_react_redux/hooks/use_store_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:over_react/over_react_redux.dart';
+import 'package:redux/redux.dart';
+import 'package:test/test.dart';
+
+import '../../test_util/test_util.dart';
+import 'package:over_react/src/over_react_redux/test_fixtures/counter_fn.dart';
+import 'package:over_react/src/over_react_redux/test_fixtures/store.dart';
+
+import 'utils.dart';
+
+main() {
+  group('useStore hook', () {
+    test('subscribes to the store and redraws when it updates', () async {
```

Comment:
This component also uses useSelector, which causes it to redraw, so this test can't actually test if useStore makes the component redraw.